javascript/javascript3/week3/shopping-cart/shopping.js

```diff
@@ -0,0 +1,99 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
+      convertToCurrency(currency){
+        switch(currency){
+          case "USD": this.price *= 0.16
+          break;
+          case "EUR": this.price *= 0.13
+          break;
+          case "ARS": this.price *= 14,53
+        }
+        return this.price
+
+      }
+  }
+  
+class ShoppingCart {
+    constructor() {
+      this.products = [];
+    }
+  
+    addProduct(product) {
+      shoppingCart.products.push(product)
+    }
+
+    removeProduct(product) {
+      for( let i = 0; i< shoppingCart.products.length; i++){
+        if(shoppingCart.products[i].name === product.name){
+          shoppingCart.products.splice(i,1)
+        }
+      }
```

Comment:
If you could have make a button and add addEventlistener to it then it will be better. Try it .

Above function should work, if u can use filter then it will be much simpler.
this.products = this.products.filter(item => item !== product);